cmd/commands/root.go

```diff
@@ -0,0 +1,40 @@
+package command
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	KubeConfig           string
+	KubeContext          string
+	OperatorNamespace    string
+	CephClusterNamespace string
```

Comment:
How about the storage cluster namespace? This will be the same as the ceph namespace.
```suggestion
	StorageClusterNamespace string
```

Reply:
right